src/Raven.Server/Commercial/LicenseValidator.cs

```diff
@@ -87,21 +91,21 @@ private static byte[] GetBytesFromBase64String(string str)
                 ms.SetLength(attributesLen);
 
                 using (var binaryWriter = new BinaryWriter(ms))
-                using (var rsa = RSA.Create())
                 {
                     binaryWriter.Write(licenseKey.Id.ToByteArray());
                     binaryWriter.Write(licenseKey.Name);
 
-                    rsa.ImportParameters(rsaParameters);
+                    RsaKeyParameters publicKey = new(false,
```

Comment:
can you use named param? It is hard to understand in GH what this 'false' is for